kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerHttpException.java

```diff
@@ -233,8 +233,9 @@ private Map<String, Object> getErrorBodyAs(Retrofit retrofit) {
         retrofit.responseBodyConverter(Map.class, new Annotation[0]);
     try {
       return converter.convert(retrofit2Response.errorBody());
-    } catch (IOException e) {
-      throw new RuntimeException(e);
+    } catch (Exception e) {
+      log.debug("Exception occurred while parsing the HTTP error response body : {}", e);
```

Comment:
```suggestion
      log.debug("Exception occurred while parsing the HTTP error response body", e);
```
but also, please include the url and response code from the response to make the log message more useful.  This means adding a new argument to the method.

Reply:
Modified the log message to print the status code , url and stack trace